keps/0020-manual-plan-execution.md

```diff
@@ -173,8 +173,55 @@ Does not look like people are really doing this that much (they usually rather f
 
 #### As a stateful service operator (person) I want to be able to run backup on my stateful service when a backup plan is defined.
 
-### Implementation Details/Notes/Constraints [optional]
+### Implementation Details/Notes/Constraints
 
-TODO: choose a solution
+We've decided to go with `Have a separate CRD just for maintaining commands to execute a plan` solution.
```

Comment:
Did we? 😊

Reply:
lol I don't know anything anymore :D 🤷‍♀ :D